### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by jesusebzcp**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
 function App() {
-  const [count, setCount] = useState(0)
-
-  return (
-
-    <h1>Very good</h1>
+  const [count, setCount] = useState(0);
 
-  )
+  return <h1>FROM PULL REQUEST</h1>;
```
Comment (ChrisDietrich405) [inline src/App.jsx:8]: very good change to the code. I LIKE IT!

Comment (jesusebzcp) [reply src/App.jsx:8]: cool

Comment (jesusebzcp) [reply src/App.jsx:8]: co

Comment (jesusebzcp) [reply src/App.jsx:8]: super

**Pull request merged**